src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -667,11 +672,26 @@
 helps['network asg'] = """
     type: group
     short-summary: Manage application security groups.
+    long-summary: You can configure network security as a natural extension of an application’s structure, ASG allows
+     you to group virtual machines and define network security policies based on those groups. You can specify an
+      application security group as the source and destination in a NSG security rule. For more information
+       visit https://docs.microsoft.com/en-us/azure/virtual-network/create-network-security-group-preview
 """
 
 helps['network asg create'] = """
     type: command
     short-summary: Create an application security group.
+    long-summary: You can configure network security as a natural extension of an application’s structure, ASG
```

Comment:
Cut-and-paste of the group description. The long description should either be part of the `asg` or `asg create` documentation, but not both.